### DISCUSSION TITLE
mspider/user-guide/service-list/index

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3708] [Announcements]
# mspider/user-guide/service-list/index

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-list/index.html

**Comment by nicole-lihui** [comment]
上述状态中有告警项, 但是当前已经废弃该状态

**Reply by nicole-lihui** [reply]
异常描述也需要更新
